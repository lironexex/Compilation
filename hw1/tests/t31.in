// comments 10 - unclosed multi-line comment
/* this is a closed multi-line comment */
/* but this multi line comment is unclosed...
