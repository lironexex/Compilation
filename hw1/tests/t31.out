1 COMMENT 1
2 COMMENT 1
Error unclosed comment
